core/src/main/java/org/openstack4j/model/network/SubPort.java

```diff
@@ -0,0 +1,28 @@
+package org.openstack4j.model.network;
+
+import org.openstack4j.common.Buildable;
+import org.openstack4j.model.common.Resource;
+import org.openstack4j.model.network.builder.SubPortBuilder;
+
+/**
+ * A network subport used ONLY in trunks
```

Comment:
Provided this `SubPort` abstraction is only applicable for `Trunk`s. It should be made apparent by either class name (`TrunkSubPort`?), making it a inner class of `Trunk` or moving it to a dedicated package.